_includes/header.html

```diff
@@ -61,15 +62,16 @@
 						</hgroup>
 
 						<div class="download">
-	  						<a href="{{ site.tgmpa.github }}/archive/{{ site.tgmpa.version }}.zip" class="latest-zip">
-								<img border="0" width="90" alt="Download latest release zipfile" src="{{ site.tgmpa.zipimage }}">
-							</a>
-	  						<a href="{{ site.tgmpa.github }}/archive/{{ site.tgmpa.version }}.tar.gz" class="latest-tar">
-								<img border="0" width="90" alt="Download latest release tarball" src="{{ site.tgmpa.tarimage }}">
-							</a>
+							<p class="download-icon">
+								<a href="{{ '/download/' | prepend: site.tgmpa.url }}" title="Download the latest release">
```

Comment:
Okidoki, this does mean the icon will be an image, not a font-icon, but I'm ok with that. I will leave in the title as well as for those users who don't recognize the icon and don't use screen readers, it gives a semi-useful mouse-over comment.